### DISCUSSION TITLE
Query performance vs. the AWS SDK?

### DISCUSSION THREAD

**Discussion by mcealera** [dynamoose/dynamoose#1352] [General]
I've been testing Dynamoose query performance vs the standard AWS SDK and noticed that Dynamoose query is always slower, sometimes taking even 2x the time. I understand some performance overhead can be expected with libraries such as Dynamoose, as I'm basically trading in performance for convenience, but I was wondering if the differences seen here are to be expected? 

![Screenshot 2022-01-28 at 11 31 22](https://user-images.githubusercontent.com/20559398/151531939-5f0c4c47-00f8-44dc-abbb-3024c28ea092.png)

Testing scenario:
* DDB table with ~2000 items
* Dynamoose running via Lambda, nodejs12
* Executing 3 functions at the same time: Get via Dynamoose(green), Query via Dynamoose(blue), Query via the SDK(orange). The query itself is the same and it returns only one item.
* Metrics retrieved via CloudWatch Lambda Insights and show the Lambda execution time (~20ms for the SDK and ~45+ for Dynamoose)

Thanks

**Comment by fishcharlie** [comment]
Duplicate of #1260.

TLDR: fixed in v3.
